src/Neo/UInt160.cs

```diff
@@ -31,68 +32,72 @@ public class UInt160 : IComparable<UInt160>, IEquatable<UInt160>, ISerializable
         /// <summary>
         /// Represents 0.
         /// </summary>
-        public static readonly UInt160 Zero = new();
+        public static UInt160 Zero => new();
```

Comment:
Slower, this create a new instance for each use

Reply:
yes, the reason for this is because you can serialize or deserialize `UInt160.Zero`. When doing so will change `UInt160.Zero` to something else. So this will fix that from happing.

Reply:
I remembered that this change was not required, `UInt160` is read only class, so it's safe and faster if we use a `static readonly` instance

Reply:
Than I'll change it back.